hedy_translation.py

```diff
@@ -5,8 +5,7 @@
 import yaml
 from os import path
 
-# TODO, FH mar 2022, why don't we grab ALL_KEYW_LANGS from hedy?
-KEYWORD_LANGUAGES = ['en', 'nl', 'es', 'fr', 'ar', 'tr', 'hi', 'nb_NO']
+KEYWORD_LANGUAGES = list(hedy.ALL_KEYWORD_LANGUAGES.keys())
 
```

Comment:
Nice TODO fix 😊 
Compared to the old list, this new list is missing 'tr', is this wanted?

Reply:
Indeed! It seems `tr` was also added only in one place, will fix!

Reply:
Fixed!